src/test/java/com/tvtracker/enterprise/EnterpriseApplicationTests.java

```diff
@@ -20,28 +20,29 @@
 
     final String TEST_USERNAME = "testUser";
     final String TEST_USER_PASSWORD = "testPassword";
+    MediaEntry mediaEntry = new MediaEntry();
+    UserAccount userAccount = new UserAccount();
```

Comment:
Instantiated both as class level variables so that you don't have to Instantiate an object each time you want to use it in a test, this really helps clean up code and stops duplication.
https://www.baeldung.com/java-clean-code

Reply:
Usually this is good... just be sure that you're not unintentionally carrying data from one method to another.  That has happened to me before.